Assets/Scripts/FeedingSystem/DogFoodBowl.cs

```diff
@@ -0,0 +1,77 @@
+﻿/*
+ * Author: Timothy Ng
+ * Description: Handles the feeding dog code through calling FeedDogs
+ */
+
+using UnityEngine.UI;
+
+
+public class DogFoodBowl : MonoBehaviourExtended
+{
+    #region Instance Accessors
+
+    public float FeedTimerLeft
+    {
+        get
+        {
+            return feedingTimer.TimeRemaining;
+        }
+    }
+
+    public bool IsCurrentlyFeeding
+    {
+        get
+        {
+            return feedingTimer.TimeRemaining != 0;
+        }
+    }
+
+    #endregion
+
+    static PPTimer feedingTimer = null;
```

Comment:
Statics should go before instance vars. The order goes
- constants 
- statics
- properties / variables (sorted by access level)

So properties and variables are intermixed, and everything should be sorted by access level